front/app/containers/Admin/pagesAndMenu/containers/EditPageForm/EditPageFormNotInNavbar.tsx

```diff
@@ -25,10 +25,15 @@ import useRemoteFiles from 'hooks/useRemoteFiles';
 import usePage from 'hooks/usePage';
 import usePageSlugs from 'hooks/usePageSlugs';
 
+interface Props {
+  hideSlugInput: boolean;
+}
+
 const EditPageFormNotInNavbar = ({
+  hideSlugInput = false,
   params: { pageId },
   intl: { formatMessage },
-}: WithRouterProps & InjectedIntlProps) => {
+}: Props & WithRouterProps & InjectedIntlProps) => {
```

Comment:
Feel free to factor WithRouter out here! Can be replaced by useParams:

`const { pageId } = useParams() as { pageId: string };
`

Reply:
This file is going to be removed in i2. So, not sure it's worth it :thinking: